test/server/GraknRunnerBase.java

```diff
@@ -136,7 +136,7 @@ public void start() {
             System.out.println("Database directory will be at " + dataDir.toAbsolutePath());
             graknProcess = executor.command(command()).start();
 
-            Thread.sleep(15000);
```

Comment:
I actually didn't notice that `master` had recently been updated to a 15 second startup time. This seems heavy-handed given the problem is only in Cluster, and it is quite uncommon.